exporter/exporterhelper/internal/persistent_queue.go

```diff
@@ -86,26 +83,31 @@ var (
 	errWrongExtensionType = errors.New("requested extension is not a storage extension")
 )
 
+type PersistentQueueSettings[T any] struct {
+	QueueSettings[T]
```

Comment:
Don't embed these, not worth for 2 members.

Reply:
done